src/main/java/database/controller/DatabaseServiceRestController.java

```diff
@@ -0,0 +1,109 @@
+package database.controller;
+
+import database.entity.BaseEntity;
+import database.service.DatabaseService;
+import database.service.JsonDatabaseService;
+import io.javalin.config.JavalinConfig;
+import io.javalin.http.BadRequestResponse;
+import io.javalin.http.Context;
+import io.javalin.http.NotFoundResponse;
+
+import java.util.List;
+import java.util.Map;
+
+import static io.javalin.apibuilder.ApiBuilder.delete;
+import static io.javalin.apibuilder.ApiBuilder.get;
+import static io.javalin.apibuilder.ApiBuilder.path;
+import static io.javalin.apibuilder.ApiBuilder.post;
+import static io.javalin.apibuilder.ApiBuilder.put;
+
+public class DatabaseServiceRestController {
+    private static final DatabaseService databaseService = new JsonDatabaseService();
```

Comment:
I think it would be good to add constructor with parameter instead of hardcoding instance.